invalidPropsals.ts

```diff
@@ -0,0 +1,122 @@
+import snapshot from '@snapshot-labs/snapshot.js';
+const dotenv = require('dotenv');
+dotenv.config();
+global['fetch'] = require('node-fetch');
+
+import db from './server/helpers/mysql';
+import { jsonParse } from './server/helpers/utils';
+import { spaces } from './server/helpers/spaces';
+import { archiveProposal, loadSpaces } from './server/helpers/adapters/mysql';
+
+const tableName = 'messages';
+const backuptableName = 'messages_backup';
+
+
+const init = async () => {
+    loadSpaces().then(async ensSpaces => {
+        console.log('ENS spaces', Object.keys(ensSpaces).length);
+
+        await backup();
+        await removeInvalidProposals();

```

Comment:
This assume we will execute this function just once, but there is ~7K proposals it will take a while to calculate score for all these. We should have some kind of cursor so we can start the process again where it stoped. Also i think it will i would be better if the verification are done for all proposals of a space at once. Instead of verifying proposal per proposal we can verify space per space that will be much faster.